### ISSUE TITLE
Notifications are not shown if connected to local network

### ISSUE THREAD

**Issue by dominicdesu** [openhab/openhab.android#253]
Hi,
if I am connected to my local LAN, the notification list in the android app is empty. However, if I am connected via mobile data (using the openHAB Remote URL), I can see the notifications.

I am using HABDroid 1.8.0.5 and the openHAB 2 snapshot from 2016-11-19.
Any idea what could be wrong?

Edit: I guess it is related to #251. Seems like my raspberrypi now receives the GET request for the notifications. I assume this should still go to my.openhab.org instead?

**Comment by cribskip** [comment]
Hi,

just found the same behavior and wanted to second this issue.
I'm using the stable debian repository and habdroid 1.8.0.5.

**Comment by eidi98** [comment]
Hi,

the same issue on my side. 
My habdroid version is 1.8.0.6

**Comment by id628** [comment]
Now the entire notifications seems to be gone, it's not in my Hamburger/Fly-out menu anymore whether I'm on the LAN or remote on 0.9.0.b5

Not sure if notifications themselves stopped working, but I can't get to the list manually.

**Comment by dominicdesu** [comment]
I can confirm, no notifications in the hamburger menu on 1.8.0.8.

**Comment by FlorianSW** [comment]
Working on it.

**Comment by magd1978** [comment]
I think I know why notifications are not received in the local network. I used to have a login / password for the reverse proxy nginx (local access) and it was the same credentials for myopenhab. I added new users to myopenhub to send different messages to different users, and changed  in app the login / password for remote access to one of the newly created ones. Now I get the messages addressed to the new user when I'm connected remotely, and those addressed to the old user when I connect locally. When local access is available, the app is using the local login / password to receive messages from the cloud, and I think that this is wrong (no local credentials, no messages). The app should always use the credentials for remote access.
